config/services/s3_idrive.toml

```diff
@@ -0,0 +1,11 @@
+[repository]
+repository = "opendal:s3"
+password = "password"
+
+[repository.options]
+root = "/"
+bucket = "bucket_name"
+endpoint = "https://p7v1.ldn.idrivee2-40.com"
+region = "gb-ldn"
```

Comment:
@kapitainsky should this be `auto` or is this working for you?

Reply:
both `auto` and explicit work. I tested. Explicit is always better to use as otherwise your requests are delayed to determine correct region.

Reply:
ok, added that as a comment